docs/root/configuration/http/http_filters/cache_filter.rst

```diff
@@ -0,0 +1,10 @@
+.. _config_http_filters_cache:
+
+Pluggable HTTP Cache
+====================
+Pluggable HTTP cache filter is not yet ready for deployment.
+
+* `Design doc <https://docs.google.com/document/d/1WPuim_GzhfdsnIj_tf-fIeutK0jO4aVQfVrLJFoLN3g/view>`_
```

Comment:
I think this PR is the right time to move the solidified design document to RST. Otherwise it might be confusing to readers which parts are plan-of-record, which bits are under debate. Generally prefer Envoy docs to be self-contained and not rely on pointer to Google Docs.

Reply:
Moved user-config overview to RST. I also want to add a separate "how to make a cache storage plugin" doc. Is there precedent to tell me where to put it?

Reply:
Probably https://github.com/envoyproxy/envoy/tree/master/source/docs.

Reply:
done